packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -273,7 +272,8 @@ export abstract class FluidDataStoreContext
 	private lastUsedRoutes: string[] | undefined;
 
 	public readonly id: string;
-	private readonly _containerRuntime: ContainerRuntime;
+	private readonly _containerRuntime: IContainerRuntimeBase;
```

Comment:
Why is this still needed? It looks like all usages were switched to using `parentContext`

Reply:
We still expose a `IContainerRuntimeBase` as part of the parentContext's interface.